configs/settings.env.sample

```diff
@@ -0,0 +1,150 @@
+# This is a **sample** of ``mailsrv``'s settings file.
```

Comment:
Add the relevant references to Postfix's/Dovecot's documentation to every parameter, so that it can be looked up without opening the targeted file.